src/Illuminate/Redis/Connectors/PredisConnector.php

```diff
@@ -42,6 +42,10 @@ public function connectToCluster(array $config, array $clusterOptions, array $op
     {
         $clusterSpecificOptions = Arr::pull($config, 'options', []);
 
+        if (! empty($config['prefix'])) {
```

Comment:
empty is not the right function here. it will not permit using 0 as a prefix.

Reply:
I simply copied the code from https://github.com/laravel/framework/blob/a049279178cfea6d8e714dcbb09429b5b78603c9/src/Illuminate/Redis/Connectors/PhpRedisConnector.php#L94

Do we want to also fix the code within PHPRedis as well?